src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/MålgruppeValidering.kt

```diff
@@ -27,4 +34,96 @@ object MålgruppeValidering {
             "målgruppe=$målgruppeType er ikke gyldig for $stønadstype"
         }
     }
+
+    fun vurderAldersvilkår(
+        vilkårperiode: LagreVilkårperiode,
+        stønadstype: Stønadstype,
+        grunnlagsdata: Grunnlagsdata,
+    ): VurderingAldersVilkår {
+        val fødselsdato = grunnlagsdata.grunnlag.fødsel?.fødselsdato
+
+        feilHvis(fødselsdato == null) { "Kan ikke vurdere aldersvilkår uten å vite fødselsdato til bruker" }
+
+        val gyldig: SvarJaNei? =
+            when (stønadstype) {
+                Stønadstype.BARNETILSYN, Stønadstype.LÆREMIDLER ->
+                    when (vilkårperiode.type as MålgruppeType) {
+                        MålgruppeType.AAP, MålgruppeType.NEDSATT_ARBEIDSEVNE, MålgruppeType.UFØRETRYGD ->
+                            vurderAldersvilkårForNedsattArbeidsevne(fødselsdato, vilkårperiode)
+                        MålgruppeType.OMSTILLINGSSTØNAD -> vurderAldersvilkårForOmstillingsstønad(fødselsdato, vilkårperiode)
+                        MålgruppeType.OVERGANGSSTØNAD -> SvarJaNei.JA
+                        MålgruppeType.DAGPENGER -> null
+                        MålgruppeType.SYKEPENGER_100_PROSENT -> null
+                        MålgruppeType.INGEN_MÅLGRUPPE -> null
+                    }
+
+                Stønadstype.BOUTGIFTER -> TODO("Stønadstype BOUTGIFTER er ikke implementert")
+            }
+        feilHvis(gyldig == SvarJaNei.NEI) {
+            "Aldersvilkår er ikke oppfylt ved opprettelse av målgruppe=${vilkårperiode.type} for behandling=${grunnlagsdata.behandlingId}"
+        }
+
+        return VurderingAldersVilkår(
+            gyldig,
+            genererFaktaSomJson(fødselsdato = fødselsdato, vilkårperiode = vilkårperiode),
+            "gitHash",
+            LocalDateTime.now(),
+        )
+    }
+
+    private fun genererFaktaSomJson(
```

Comment:
Denne "JSON-strukturen" blir ikke helt riktig, må rettes på

Reply:
Hvorfor trenger vi å ha med vilkårperiode her?

Reply:
vilkårperiode inngår i vurderingen av aldersvilkår. Jeg har tenkt at den derfor må inkluderes som JSON i `inputFakta` på `AutomatiskVurdering`. Dette for å være i henhold til etterlevelse for "automatisk vurdering".
Med andre ord: Vi skal kunne vite hvilke fakta som lå til grunn når det ble utført en "automatisk vurdering"

Reply:
Men ettersom `fom` og `tom` allerede blir lagret på vilkårsperiode-raden i databasen så er det vel kanskje ikke nødvendig å ta den med i `inputFakta`. Ref din andre kommentar her så skal vi vel ta en runde med teamet på hva som er nødvendig å lagre spesifikt som `inputFakta`